src/de/dkfz/b080/co/aceseq/ACESeqMethods.groovy

```diff
@@ -27,10 +27,7 @@ final class ACESeqMethods {
     static LinkedHashMap<String, String> getGlobalJobSpecificParameters(Configuration config) {
         return new LinkedHashMap<String, String>(
                 (ACEseqConstants.CHR_NAME): config.configurationValues.getString(ACEseqConstants.CHR_NAME),
-                (ACEseqConstants.CHR_NR): config.configurationValues.getString(ACEseqConstants.CHR_NR),
-                (ACEseqConstants.GENETIC_MAP_FILE): config.configurationValues.getString(ACEseqConstants.GENETIC_MAP_FILE),
-                (ACEseqConstants.KNOWN_HAPLOTYPES_FILE): config.configurationValues.getString(ACEseqConstants.KNOWN_HAPLOTYPES_FILE),
-                (ACEseqConstants.KNOWN_HAPLOTYPES_LEGEND_FILE): config.configurationValues.getString(ACEseqConstants.KNOWN_HAPLOTYPES_LEGEND_FILE))
+                (ACEseqConstants.CHR_NR): config.configurationValues.getString(ACEseqConstants.CHR_NR))
```

Comment:
In phasing.sh and phasing_X.sh `CHR_NR` is not used but overwritted by a local definition -- composed of the prefix and the name. Mabe this variable here should be used there?

Reply:
I am not familiar with the groovy code and how these are read into the shell script. The `CHR_NAME` used here is also present in the shell script. Could you provide a commit suggestions, I will add them.